widgets/price-table/js/pricetable.js

```diff
@@ -1,4 +1,28 @@
 jQuery( function($){
+    $('.ow-pt-columns-atom.sow-equalize-row-heights').each(function () {
```

Comment:
In this case, if we targetted `sow-equalize-row-heights` (along with something else to make sure we only target the price table), we could make it up to the template to decide if it wants equalised heights. Would something like that work? It seems better than adding each individual class to this selector.